tests/tests_integration/test_api/test_simulators/seed/data.py

```diff
@@ -0,0 +1,740 @@
+simulator = {
+    "name": "DWSIM",
+    "externalId": "integration_tests_workflow",
+    "fileExtensionTypes": ["dwxmz"],
+    "modelTypes": [{"name": "Steady State", "key": "SteadyState"}],
+    "stepFields": [
+        {
+            "stepType": "get/set",
+            "fields": [
+                {
+                    "name": "objectName",
+                    "label": "Simulation Object Name",
+                    "info": "Enter the name of the DWSIM object, i.e. Feed",
+                },
+                {
+                    "name": "objectProperty",
+                    "label": "Simulation Object Property",
+                    "info": "Enter the property of the DWSIM object, i.e. Temperature",
+                },
+            ],
+        },
+        {
+            "stepType": "command",
+            "fields": [
+                {
+                    "name": "command",
+                    "label": "Command",
+                    "info": "Select a command",
+                    "options": [{"label": "Solve Flowsheet", "value": "Solve"}],
+                }
+            ],
+        },
+    ],
+    "unitQuantities": [
+        {
+            "name": "area",
+            "label": "Area",
+            "units": [{"label": "m2", "name": "m2"}, {"label": "cm2", "name": "cm2"}, {"label": "ft2", "name": "ft2"}],
+        },
+        {
+            "name": "head",
+            "label": "Head",
+            "units": [{"label": "m", "name": "m"}, {"label": "ft", "name": "ft"}, {"label": "cm", "name": "cm"}],
+        },
+        {
+            "name": "mass",
+            "label": "Mass",
+            "units": [{"label": "kg", "name": "kg"}, {"label": "g", "name": "g"}, {"label": "lb", "name": "lb"}],
+        },
```

Comment:
we can leave 3-5 of them, e.g. temperature, speed, volume, etc.
 this is too many for the sake of the test